docs/src/modules/components/MaterialYouUsageDemo.tsx

```diff
@@ -93,14 +90,6 @@ export const prependLinesSpace = (code: string, size: number = 2) => {
   return newCode.join('\n');
 };
 
-function ModeSwitcher({ md2Mode }: { md2Mode: Mode }) {
-  const { setMode } = useColorScheme();
-  React.useEffect(() => {
-    setMode(md2Mode);
-  }, [md2Mode, setMode]);
-  return null;
-}
-
```

Comment:
Harmful, either we use different `modeStorageKey` and `attribute` keys between Material UI and Material UI Next or we remove this.